faqs/galaxy/datatype_tabular.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Identifying and formatting Tabular Datasets 
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+contributors: [jennaj, beachyesh]
+layout: faq          
```

Comment:
I'm not sure the order of the rows here matters, but just for consistency, could you move the contributors to the end? I think that's how it appears in the other FAQs. Thanks!

Reply:
Done. Thanks!